tests/etl/test_loaders.py

```
@@ -0,0 +1,59 @@
+
+    def execute_query(self, *args, **kwargs):
+        pass
+
+    def commit_transaction(self):
+        pass
+
+    def rollback_transaction(self):
+        pass
+
+
+def test_postgresql_loader_basic_insert():
```

Ruff (0.11.9)

Missing return type annotation for public function `test_postgresql_loader_basic_insert`

Add return type annotation: `None`

(ANN201)